### PR TITLE
File Upload Dark Mode - RSC-1521 RSC-1524

### PR DESCRIPTION
**Pull request by rpokorny**
https://issues.sonatype.org/browse/RSC-1521
https://issues.sonatype.org/browse/RSC-1524

`NxMultiFileUpload` is included here as it turned out to require no additional work aside from committing its snapshots.

### CODE REVIEW

**Comment by rpokorny** [comment]
Converting back to draft as I noticed the border thicknesses are different on dark mode, which I have not yet implemented.

File: lib/src/base-styles/_nx-variables.scss [hunk 1]
```diff
@@ -65,17 +65,18 @@
...
   }
 
   // Deprecated variables
   --nx-border-subsection: 1px solid var(--nx-color-subsection-border);
   --nx-box-shadow-focus: 0 0 3px 1px var(--nx-color-interactive-shadow-focus);
   // similar to box-shadow-focus listed above, but for use in drop-shadow filters rather than box-shadow properties
   --nx-drop-shadow-focus: 0 0 3px var(--nx-color-interactive-shadow-focus);
-
-  @include nx-dark-mode-helpers.dark-mode {
-    --nx-border-width-button: 2px;
```
Comment (rpokorny) [inline lib/src/base-styles/_nx-variables.scss:79]: Moved into the dark mode block above

**Comment by cindyko226** [comment]
Missing new variables documentation.

**Comment by rpokorny** [comment]
> Missing new variables documentation.

Thanks. 8be5b12b8

**Review by cindyko226 — APPROVED** [review]
LGTM

**Pull request merged**